subsys/net/lib/lwm2m/ipso_pressure_sensor.c

```diff
@@ -83,7 +83,7 @@ static void update_max_measured(uint16_t obj_inst_id, int index)
 	NOTIFY_OBSERVER(IPSO_OBJECT_ID, obj_inst_id, MAX_MEASURED_VALUE_RID);
 }
 
-static int reset_min_max_measured_values_cb(uint16_t obj_inst_id)
+static int reset_min_max_measured_values_cb(uint16_t obj_inst_id, uint8_t *payload, uint16_t payload_len)
```

Comment:
Plus there are many more callbacks that would need an update, a quick grep for `INIT_OBJ_RES_EXECUTE` shows the following files:
* ipso_generic_sensor.c
* ipso_humidity_sensor.c
* lwm2m_obj_device.c
* lwm2m_obj_server.c